src/screens/ledger/ledger-connect-component.tsx

```diff
@@ -240,24 +243,45 @@ export class LedgerConnectComponent extends React.Component<
     }
 
     private async selectStep(step: ScreenStep) {
-        await this.stepContainerFadeOut();
-        this.setState({ step });
+        await this.stepContainerFadeOut(step);
+        this.setState({ step, stepContainerTranslateAnimation: new Animated.Value(400) });
         await this.stepContainerFadeIn();
     }
 
     private async stepContainerFadeIn() {
-        Animated.timing(this.stepContainetFadeAnimation, {
-            toValue: 1,
-            duration: FADE_ANIMATION_TIME
-        }).start();
+        Animated.parallel([
+            Animated.timing(this.state.stepContainerFadeAnimation, {
+                toValue: 1,
+                duration: FADE_ANIMATION_TIME,
```

Comment:
since this is not a fade animation anymore, it's confusing to have this name. Change it to a more generic name `ANIMATION_TIME` for example.